public/app/features/inspector/InspectStatsTab.tsx

```diff
@@ -59,11 +60,14 @@ export const InspectStatsTab = ({ data, timeZone }: InspectStatsTabProps) => {
 
   const statsTableName = t('dashboard.inspect-stats.table-title', 'Stats');
   const dataStatsTableName = t('dashboard.inspect-stats.data-title', 'Data source stats');
+  const traceIdsStatsTableName = t('dashboard.inspect-stats.data-traceids', 'Trace ids');
```

Comment:
Not sure about this.

```suggestion
  const traceIdsStatsTableName = t('dashboard.inspect-stats.data-traceids', 'Trace IDs');
```